cmd/ebrelayer/contract/generated/bin/Valset/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
can we replace these all with rules in our base .gitignore file??

Reply:
probably yeah - want to ignore the .bin and .abi files but not the directories, if I recall the generator throws an err when it can't find the proper directory to write to. Could also update the generator instead if global .gitignore rules fall short.